### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by Patryk27**
Closes https://github.com/nix-community/naersk/issues/283.

### CODE REVIEW

**Comment by Patryk27** [comment]
Fair point! Approach #2, lemme know what you think 🙂

**Review by nmattia — APPROVED** [review]
Love it! Do you think it makes sense to create a new section, like "Introduction" or similar?

File: README.md [hunk 1]
```diff
@@ -9,6 +9,34 @@ Build Rust projects with ease!
 * [Examples](#examples)
 * [Tips & Tricks](#tips--tricks)
 
+Naersk is a Nix library for building Rust projects - basically, you write:
```
Comment (nmattia) [inline README.md:12]:
```suggestion
Naersk is a [Nix](https://nixos.org/explore.html) library for building Rust projects - basically, you write:
```

**Comment by Patryk27** [comment]
Sure, I was thinking about that myself, just wasn't sure - let's see it now :smile:

File: README.md (hunk 1 above)
Comment (Patryk27) [reply README.md]: kk, added

**Review by nmattia — APPROVED** [review]
❤️

**Pull request merged**